docs/en/entity-onboarding.rst

```diff
@@ -0,0 +1,1109 @@
+
+**Step 2 - Federation Authority Validation and Certificate Issuance**: Following the onboarding request submission, the **Federation Authority** MUST perform:
+
+  - Verification of information provided in the registration request.
+  - Validation of the Entity Configuration, and the metadata contained in it, published at the entity's ``/.well-known/openid-federation`` endpoint (as defined in :ref:`trust:The Infrastructure of Trust`).
+  - **Metadata Policy Application**: Application of federation-specific metadata policies to the entity's metadata based on organizational characteristics and authorization scope as defined in :ref:`trust:Subordinate Statements`. When onboarded through an Intermediate, both Intermediate and Trust Anchor policies apply, with Trust Anchor policies taking precedence in case of conflicts.
+  - **Certificate Issuance**: Certification of the Federation Entity Public Key with X.509 Certificate issuance using the trust infrastructure detailed in :ref:`trust:Trust Infrastructure Requirements`. Intermediates issue Certificates with appropriate **naming constraints** to scope Certificate usage to their subordinates only.
+
+Upon successful validation, the entity receives a response containing an X.509 Certificate Chain where:
+
+  - The first element is the X.509 Certificate that certifies the Federation Entity Public Key (issued by the Federation Authority).
+  - **For Trust Anchor onboarding**: The second element is the Trust Anchor's self-signed X.509 Certificate for validating the first X.509 Certificate.
+  - **For Intermediate onboarding**: Additional elements include the Intermediate's X.509 Certificate and the Trust Anchor's self-signed Certificate, forming a complete X.509 Certificate chain.
+  - All X.509 Certificates are expressed in DER format encoded in Base64.
+
+Example X.509 Certificate chain response:
+
+.. code-block:: json
+
+   [
+     "MIIDqjCCA1GgAwIBAgIGAZc6/V9qMAoGCCqGSM49BAMCMIGzMQsw...",
+     "MIIDQzCCAuigAwIBAgIGAZc6+XlDMAoGCCqGSM49BAMCMIGzMQsw..."
+   ]
+
+.. note::
+   If validation fails, the entity receives a response with identified issues to be resolved before submitting a new onboarding request.
+
+**Step 3 - Entity Configuration Update and Resolve Request**: After receiving the X.509 Certificate chain from the Federation Authority, the entity MUST:
+
+  1. **Update Entity Configuration**:
+
+    - Add an ``authority_hints`` claim with a JSON Array containing the **immediate Federation Authority's** Federation Entity Identifier (Trust Anchor for direct onboarding, or Intermediate for mediated onboarding) as defined in :ref:`trust:Federation Roles`.
+    - Update the Federation Entity Public Key in the ``jwks`` claim by adding an ``x5c`` claim with the complete X.509 Certificate chain received from the Federation Authority.
```

Comment:
@fmarino-ipzs this may appear as a design error, because an entity can get onboarded by more than a single TA and therefore having multiple certificates chain belonging to multiple TA, while this approach only use a single TA and this doesn't scale